server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/event/ComputeMessagePreviewListenerTest.java

```diff
@@ -0,0 +1,181 @@
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.event;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.nio.charset.StandardCharsets;
+
+import org.apache.james.core.Username;
+import org.apache.james.jmap.api.preview.MessagePreviewStore;
+import org.apache.james.jmap.api.preview.Preview;
+import org.apache.james.jmap.draft.model.message.view.MessageFullViewFactory;
+import org.apache.james.jmap.draft.utils.HtmlTextExtractor;
+import org.apache.james.jmap.draft.utils.JsoupHtmlTextExtractor;
+import org.apache.james.jmap.memory.preview.MemoryMessagePreviewStore;
+import org.apache.james.mailbox.MailboxSession;
+import org.apache.james.mailbox.MailboxSessionUtil;
+import org.apache.james.mailbox.MessageIdManager;
+import org.apache.james.mailbox.MessageManager;
+import org.apache.james.mailbox.events.Group;
+import org.apache.james.mailbox.inmemory.manager.InMemoryIntegrationResources;
+import org.apache.james.mailbox.model.ComposedMessageId;
+import org.apache.james.mailbox.model.FetchGroupImpl;
+import org.apache.james.mailbox.model.MailboxId;
+import org.apache.james.mailbox.model.MailboxPath;
+import org.apache.james.mailbox.model.MessageRange;
+import org.apache.james.mailbox.model.MessageResult;
+import org.apache.james.mailbox.store.StoreMailboxManager;
+import org.apache.james.mime4j.dom.Message;
+import org.apache.james.util.mime.MessageContentExtractor;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import reactor.core.publisher.Mono;
+
+class ComputeMessagePreviewListenerTest {
+    private static final Username BOB = Username.of("bob");
+    private static final Preview PREVIEW = Preview.from("This should be the preview of the message...");
+    private static final MailboxPath BOB_INBOX_PATH = MailboxPath.inbox(BOB);
+    private static final MailboxPath BOB_OTHER_BOX_PATH = MailboxPath.forUser(BOB, "otherBox");
+
+    private MessagePreviewStore messagePreviewStore;
```

Comment:
private and final are useless in junit classes because we consider nobody will write stupid things in tests
It allows for a better signal/noise ratio

Reply:
usually we get remarks when we forget about those, now we should not put them anymore? I'm a bit confused to be honest... any more details on why we should adopt this new coding style?

Reply:
it's not new, it's junit5 convention. It's the reason why everything is in "default" scope now.
Otherwise, it doesn't make sense to have tests methods package-private, the only rational I have is : "in tests we try to have an optimal signal/noise ratio". (see https://stackoverflow.com/a/55230350)
If you know another reason, let me know.

Reply:
Nope, just wanted to understand. Thanks for the link and the feedback :)